firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigAutoFetch.java

```diff
@@ -144,7 +146,9 @@ private void handleNotifications(InputStream inputStream) throws IOException {
             if (jsonObject.has(REALTIME_DISABLED_KEY)
                 && jsonObject.getBoolean(REALTIME_DISABLED_KEY)) {
               retryCallback.onError(
-                  new FirebaseRemoteConfigRealtimeUpdateStreamException("Realtime is disabled."));
+                  new FirebaseRemoteConfigServerException(
```

Comment:
Is this error passed to the user or just used internally to communicate state? IIUC it's just passed to the `retryCallback` created in `startAutoFetch()` below, which creates the user facing error.

Reply:
It's a mixture of both. This exception will pass state to the realtimeHttpClient and then the realtimeHttpClient will take this exact exception and propagate it up to the user.

Reply:
Got it - I see that updated now to propagate the same error passed to the callback 👍